migration-from-4x.md

```diff
@@ -334,7 +340,9 @@ editor.addButton('mybutton', {
 ```js
 editor.ui.registry.addButton('myButton', {
   text: 'My Button',
-  onAction: (_) => editor.execCommand('mceSave')
+  onAction: function (_) {
```

Comment:
Another place I'd suggest we remove the underscore, as it's more likely to cause confusion.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d